src/main/java/org/myrobotlab/opencv/OpenCVData.java

```diff
@@ -155,6 +155,8 @@ public static void main(String[] args) {
   // for use with text detection / ocr filters.
   private ArrayList<DetectedText> detectedText;
   
+  private Double blurriness = null;
+  

```

Comment:
This will go out on every opencvdata ?

Reply:
I didn't really know where else to put it.